app/utils/job-config-arch.js

```diff
@@ -0,0 +1,16 @@
+import { archConfigKeys } from 'travis/utils/keys-map';
+
+export default function jobConfigArch(config) {
+  if (!config) { return ''; }
```

Comment:
It seems like previously it has always been returning `amd64` - according to Percy diffs (please take a look). 

In that case, you could just extract variables from config right in the argument declaration:
```js
export default function jobConfigArch({ arch, os } = {}) {
```
In case when no config is present, both variable will be `undefined` and by default it'll return `amd64`.

Reply:
I guess, normally, config should not be null, but probably it makes sense